src/api/api.controller.ts

```diff
@@ -165,6 +166,22 @@ export class ApiController {
         return await this.issuerService.revokeCredential(body.credential_exchange_id, body.publish);
     }
 
+    /**
+     * Deletes a credential for the issuer
+     */
+    @Delete('issuer/records/:credExId}')
```

Comment:
There's an extra } at the end - be sure to test these endpoints locally (eg via postman or insomnia)